### PR TITLE
feat: improve help UX

### LINKED ISSUE #279: Improve help UX
Relates to https://github.com/adobe/helix-sidekick-extension/issues/67

### PR DESCRIPTION
**Pull request by rofe**
Fix #279

### CODE REVIEW

File: src/sidekick/fr.css [hunk 1]
```diff
@@ -86,8 +86,16 @@
   content: "Got it!";
 }
 
-.hlx-sk-overlay .modal.help .help-dismiss::before {
-  content: "Not now";
+.hlx-sk-overlay .modal.help .help-close-dismiss::before {
+  content: "Show help topic again later";
```
Comment (rofe) [inline src/sidekick/fr.css:90]: @kptdobe could you please provide the French translations?

File: src/sidekick/fr.css [hunk 2]
```diff
@@ -86,8 +86,16 @@
   content: "Got it!";
```
Comment (kptdobe) [inline src/sidekick/fr.css:86]:
```suggestion
  content: "J'ai compris!";
```

File: src/sidekick/fr.css (hunk 1 above)
Comment (kptdobe) [inline src/sidekick/fr.css:90]:
```suggestion
  content: "Revoir l'aide plus tard";
```

File: src/sidekick/fr.css [hunk 3]
```diff
@@ -86,8 +86,16 @@
...
 }
 
-.hlx-sk-overlay .modal.help .help-dismiss::before {
-  content: "Not now";
+.hlx-sk-overlay .modal.help .help-close-dismiss::before {
+  content: "Show help topic again later";
+}
+
+.hlx-sk-overlay .modal.help .help-close-acknowledge::before {
+  content: "Don't show help topic again";
```
Comment (kptdobe) [inline src/sidekick/fr.css:94]:
```suggestion
  content: "Ne plus voir ce sujet";
```

File: src/sidekick/fr.css [hunk 4]
```diff
@@ -86,8 +86,16 @@
...
+}
+
+.hlx-sk-overlay .modal.help .help-close-opt-out::before {
+  content: "Never show any help topics";
```
Comment (kptdobe) [inline src/sidekick/fr.css:98]:
```suggestion
  content: "Ne plus voir l'aide";
```

**Review by kptdobe — APPROVED** [review]

File: src/sidekick/fr.css [hunk 5]
```diff
@@ -83,11 +83,19 @@
...
+
+.hlx-sk-overlay .modal.help .help-close-acknowledge::before {
+  content: "Ne plus voir ce sujet";
+}
+
+.hlx-sk-overlay .modal.help .help-close-opt-out::before {
+  content: "Ne plus voir l'aide";
 }
 
 .hlx-sk-overlay .modal.help .help-previous::before {
```
Comment (rofe) [inline src/sidekick/fr.css:101]: @kptdobe thanks for the translations above! I found 3 more untranslated strings (L101 and following). They are not part of this PR but maybe you could make a direct commit to the branch?
